<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bookingDetails</class>
 <widget class="QDialog" name="bookingDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>211</width>
     <height>411</height>
    </rect>
   </property>
   <property name="title">
    <string>Buchung Details</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>203</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>ID</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_16">
           <property name="maximumSize">
            <size>
             <width>39</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>Preis</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>von</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>bis</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QFormLayout" name="formLayout_3">
         <item row="1" column="0">
          <widget class="QDoubleSpinBox" name="bookingPrice">
           <property name="maximum">
            <double>99999999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QDateEdit" name="bookingFromDate"/>
         </item>
         <item row="3" column="0">
          <widget class="QDateEdit" name="bookingToDate"/>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLineEdit" name="bookingId">
           <property name="maximumSize">
            <size>
             <width>161</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="detailsTab">
       <property name="minimumSize">
        <size>
         <width>201</width>
         <height>284</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>201</width>
         <height>291</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="trainTab">
        <attribute name="title">
         <string>Train Booking</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>1</y>
           <width>60</width>
           <height>161</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="abfahrtLabel">
            <property name="text">
             <string>Abfahrt in</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Abfahrt Time</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="ankunftLabel">
            <property name="text">
             <string>Ankunft in</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_3">
            <property name="maximumSize">
             <size>
              <width>61</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>Ankunft Time</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="ankunftLabel_4">
            <property name="maximumSize">
             <size>
              <width>69</width>
              <height>21</height>
             </size>
            </property>
            <property name="text">
             <string>Ticket Type</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="uberLabel">
            <property name="text">
             <string>Uber:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>2</y>
           <width>141</width>
           <height>251</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="4" column="0">
           <widget class="QLineEdit" name="trainTicketType">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QPlainTextEdit" name="trainConnectingStations">
            <property name="maximumSize">
             <size>
              <width>121</width>
              <height>101</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLineEdit" name="trainAnkunftTime">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLineEdit" name="trainToDestination">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="trainAbfahrtTime">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLineEdit" name="trainFromDestination">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="hotelTab">
        <attribute name="title">
         <string>Hotel Booking</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget_5">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>71</width>
           <height>71</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="abfahrtLabel_2">
            <property name="text">
             <string>Town</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="ankunftLabel_2">
            <property name="maximumSize">
             <size>
              <width>69</width>
              <height>21</height>
             </size>
            </property>
            <property name="text">
             <string>Hotel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="abfahrtLabel_4">
            <property name="text">
             <string>Room Type</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget_6">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>0</y>
           <width>121</width>
           <height>76</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLineEdit" name="townName">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="hotelName">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="hotelRoomType">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="flightTab">
        <attribute name="title">
         <string>Flight Booking</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget_7">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>71</width>
           <height>151</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="abfahrtLabel_3">
            <property name="text">
             <string>From</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="ankunftLabel_3">
            <property name="text">
             <string>To</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="uberLabel_2">
            <property name="text">
             <string>Airline</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="uberLabel_3">
            <property name="text">
             <string>Booking Class</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget_8">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>0</y>
           <width>121</width>
           <height>154</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLineEdit" name="flightFromDestination">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="flightFromAirport">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="flightToDestination">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="flightToAirport">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="flightAirline">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="flightBookingClass">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="carTab">
        <attribute name="title">
         <string>Car Rental</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget_9">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>65</width>
           <height>101</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="minimumSize">
             <size>
              <width>63</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>Pickup Location</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="minimumSize">
             <size>
              <width>63</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>Return Location</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Company</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Vehicle Class</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget_10">
         <property name="geometry">
          <rect>
           <x>75</x>
           <y>0</y>
           <width>121</width>
           <height>102</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLineEdit" name="carPickupLocation">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="carReturnLocation">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="carCompany">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="carVehicleClass">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>420</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
